piqe_ocp_lib/api/resources/ocp_operators.py

```diff
@@ -200,6 +200,19 @@ def get_package_ownnamespace_channel(self, package_name):
             logger.error("A OwnNamespace channel was not found for package: {}".format(package_name))
         return None
 
+    def get_yaml_from_annotation(self, channel_name):
+        """
+        A method that returns a channel object
+        :param channel_name: name of the channel 
+        :return: channel object
+        """
+        pkg_details = self.get_package_manifest("amq-streams")
```

Comment:
We should pass this method call the 'operator_name' argument instead of hard coding it.

Reply:
done